### DISCUSSION TITLE
Suggestion: Add media player to above volume list

### DISCUSSION THREAD

**Discussion by jac0-b** [File-New-Project/EarTrumpet#833] [Ideas]
Would be nice to have a media player above the list, similar to the media overlay. Could be pretty useful for people without media buttons on their keyboard.

![](https://user-images.githubusercontent.com/51512690/118408849-3ae32a80-b67f-11eb-8437-5ba47287fe89.png)

**Comment by hornyjailfish** [comment]
yes need this too
